### PR TITLE
Windows build

### LINKED ISSUE #1: Windows build
We need to build `freetype` on Windows to avoid pining Windows package to `2.5*` and  `*nix` packages to `2.6*`.

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #1

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+:: Test.
+ctest
+if errorlevel 1 exit 1
+
+:: Install.
+nmake install
+if errorlevel 1 exit 1
+
+:: Move everything 1-level down.
```
Comment (jakirkham) [inline recipe/bld.bat:25]: I guess it should say "1-level up".

**Comment by jakirkham** [comment]
Good work, @ocefpaf.

File: recipe/bld.bat (hunk 1 above)
Comment (jakirkham) [inline recipe/bld.bat:25]: This is not a block for me though.

**Comment by ocefpaf** [comment]
I was going to ask for @gillins to review this, but I guess you were faster.

**Comment by jakirkham** [comment]
Sorry.

**Comment by ocefpaf** [comment]
> Sorry.

Don't worry. I can iterate this in another PR. Just bare in mind that, even without the `WIP` label it is nice to let a PR sit for at a few hours. Preferably a 1-2 days due to timezone differences.

Also, as many of us here, I don't have a Windows machine to tests this, so another opinion is very important.

**Comment by jakirkham** [comment]
Well, if this is your intention, could you please add WIP in the future? As there are so many things going on here, it is tricky to determine what is important or how pressing.

**Comment by ocefpaf** [comment]
> Well, if this is your intention, could you please add WIP in the future?

Sure. I just re-enforce my request to let a PR sit a little bit regardless. Merging something that was just submitted is not desirable regardless of WIP.

**Comment by jakirkham** [comment]
I hear you, but I'm waking up with nearly 100 notifications from here every morning. Please make life simple here. A giant WIP makes this obvious.

**Comment by ocefpaf** [comment]
> I hear you, but I'm waking up with nearly 100 notifications from here every morning.

That is exactly why I am asking you to not react to all of them. There are more people in the team here BTW.

**Comment by jakirkham** [comment]
Which is why I'm asking you to help filter these. Having it be WIP let's me easily steer clear before coming to GitHub. Timestamps don't do that (nor are they as obvious once I get here).

I'm not trying to make a big thing out of this. I hear you about letting them sit and have said as much. Just asking, that we get in the practice of always adding WIP. Was merely trying to provide some perspective as to why I ask. That's all.

**Comment by ocefpaf** [comment]
> Which is why I'm asking you to help filter these. Having it be WIP let's me easily steer clear before coming to GitHub. Timestamps don't do that (nor are they as obvious once I get here).

As I said before. Sure. I am ready to comply to your way to deal with things.

What I am asking of you though is that you do not react to every PR/Issue/message here. There are teams to share the task.

See... What you ask of me adds a burden on my side and I am OK with it. What I ask of you removes a burden from your side and I guess you should think about it at least :wink:

**Comment by jakirkham** [comment]
> What I am asking of you though is that you do not react to every PR/Issue/message here. There are teams to share the task.

Again, I'll try to let them sit. Having them be WIP helps.

> See... What you ask of me adds a burden on my side and I am OK with it. What I ask of you removes a burden from your side and I guess you should think about it at least :wink:

It's a bit of a two way street. I should have added WIP to the recent Boost PR ( https://github.com/conda-forge/boost-feedstock/pull/5 ) as I suspected we needed more tests last night, but I didn't. So, now we need to add more tests that we could have added in the original PR. It's fine, but it is the same story as here with our roles reversed. Hence why I am asking we start to do this more to help each other. 😉

If you can think of a simpler way to do this that still works, I'm open to new ideas. Right now, it seems adding WIP to the title is pretty easy though. Not doing it is what seems to be expensive.

**Pull request merged**